user-service/model/user-orm.js

```diff
@@ -1,9 +1,11 @@
-import { createUser, findUserbyUsername } from './repository.js';
+import { createUser, findUserbyUsername, findOneUser } from './repository.js';
+import bcrypt from 'bcrypt'
 
 //need to separate orm functions from repository to decouple business logic from persistence
 export async function ormCreateUser(username, password) {
     try {
-        const newUser = await createUser({username, password});
+        const hash = bcrypt.hashSync(password, 10);

```

Comment:
Can you change the salt to use username and some special character?
For example for user - userabc, the salt could be - userabc!$$!
Its just more common practice to salt like this.

Reply:
Resolved using the length of username since hashsync only accepts integers to salt